test/node/bson_test.js

```diff
@@ -1,27 +1,27 @@
 'use strict';
-
-var Buffer = require('buffer').Buffer,
-  fs = require('fs'),
-  expect = require('chai').expect,
-  BSON = require('../..'),
-  Code = BSON.Code,
-  BSONRegExp = BSON.BSONRegExp,
-  Binary = BSON.Binary,
-  Timestamp = BSON.Timestamp,
-  Long = BSON.Long,
-  ObjectID = BSON.ObjectID,
-  ObjectId = BSON.ObjectID,
-  Symbol = BSON.Symbol,
-  DBRef = BSON.DBRef,
-  Decimal128 = BSON.Decimal128,
-  Int32 = BSON.Int32,
-  Double = BSON.Double,
-  MinKey = BSON.MinKey,
-  MaxKey = BSON.MaxKey,
-  BinaryParser = require('../binary_parser').BinaryParser,
-  vm = require('vm');
-
-var createBSON = require('../utils');
+const fs = require('fs');
+const { Buffer } = require('buffer');
+const { expect } = require('chai');
```

Comment:
Is this bump going to drop node 4 support? (see: travis errors)

Reply:
Yes it will. `bson-ext` v4 will track `bson` v4, and have the same version requirements.